app/views/multiyear/how_it_works.scala.html

```diff
@@ -46,8 +46,8 @@
     @p(Text(Messages("pages.how-it-works.lede-pre2", NumberFormat.getIntegerInstance().format(applicationConfig.MAX_BENEFIT()))))
     @p(Text(Messages("pages.how-it-works.lede-pre4", NumberFormat.getIntegerInstance().format(applicationConfig.PERSONAL_ALLOWANCE()))))
 
-    
-    @insetText(InsetText(content = Text(Messages("pages.how-it-works.lede-pre5", (applicationConfig.currentTaxYear.finishYear - 5).toString))))
```

Comment:
i think we need this, can we have a quick mob on it?

Reply:
Added back in with dynamic unit test to accompany the change
Thanks